peerscout_api/main.py

```diff
@@ -356,6 +379,29 @@ def get_response_json(
         }
 
 
+def write_peerscout_api_response_to_bq(
+    recommendation_request: dict,
+    recommendation_response: dict
+):
+    PROJECT_NAME = 'elife-data-pipeline'
+    TARGET_DATASET_NAME = get_target_dataset_env()
```

Comment:
would suggest to use lower case for any local variables and constants only at the module level?